public_suffix_list.dat

```diff
@@ -11052,6 +11052,20 @@ webhop.org
 worse-than.tv
 writesthisblog.com
 
+// ddnss.de : https://www.ddnss.de/
+// Submitted by Robert Niedziela <webmaster@ddnss.de> 2016-03-14
+
+ddnss.de
+ddnss.org
+dyn.ddnss.de  
+dyndns.ddnss.de  
+dyndns1.de
+dyn-ip24.de
+home-webserver.de  
+myhome-server.de  
+dyn.home-webserver.de
```

Comment:
Please keep the suffices in alphabetical order.

In this case,
home-webserver.de
dyn.home-webserver.de
myhome-server.de

(Sorting from the right-most, most generic label, to the left-most, most specific label)